pkg/trace/pb/remote_rates.go

```diff
@@ -0,0 +1,15 @@
+package pb
+
+// TargetTPS contains the targeted traces per second the agent should try to sample for a particular service and env
```

Comment:
We don't need these to be in the `pb` package right? They should be in `pkg/trace/sampler`